### PR TITLE
Updated the description of HITS

### PR DESCRIPTION
**Pull request by romanatarango**
Updated the description of HITS, added the description of the original version of the algorithm.

### CODE REVIEW

File: 3.11/graphs-pregel-algorithms.md [hunk 1]
```diff
@@ -127,26 +127,45 @@ pregel.start("scc", "graphname", { resultField: "component_strong" });
...
+[Authoritative sources in a hyperlinked environment](http://www.cs.cornell.edu/home/kleinber/auth.pdf),
+Journal of the ACM. 46 (5): 604–632, 1999. The algorithm is also known as _Hubs and Authorities_.
 
-The idea behind Hubs and Authorities comes from the typical structure of the web:
-Certain websites known as hubs, serve as large directories that are not actually
-authoritative on the information that they hold. These hubs are used as
-compilations of a broad catalog of information that leads users direct to other
-authoritative webpages.
+The idea behind hubs and authorities comes from the typical structure of the early web:
+Certain websites, known as hubs, serve as large directories that are not actually
```
Comment (nerpaula) [inline 3.11/graphs-pregel-algorithms.md:135]:
```suggestion
certain websites, known as hubs, serve as large directories that are not actually
```

File: 3.11/graphs-pregel-algorithms.md [hunk 2]
```diff
@@ -127,26 +127,45 @@ pregel.start("scc", "graphname", { resultField: "component_strong" });
...
 hub score. The authority score of a vertex rates the total hub score of vertices
-pointing to that vertex; the hub score rates the total authority score of vertices pointed by it. Also see
-[en.wikipedia.org/wiki/HITS_algorithm](https://en.wikipedia.org/wiki/HITS_algorithm){:target="_blank"}
-
-ArangoDB's version of the algorithm converges after a certain amount of time.
-The `threshold` parameter can be used to set a limit for the convergence
+pointing to that vertex; the hub score rates the total authority 
+score of vertices pointed by it. Also see
+[en.wikipedia.org/wiki/HITS_algorithm](https://en.wikipedia.org/wiki/HITS_algorithm){:target="_blank"}.
+(Note,however, that this version of the algorithm is slightly different from that of the original paper.)
```
Comment (nerpaula) [inline 3.11/graphs-pregel-algorithms.md:145]:
```suggestion
Note, however, that this version of the algorithm is slightly different from that of the original paper.
```

**Review by nerpaula — COMMENTED** [review]
LGTM. Just few minor things. 
Needs to be added to 3.10 as well.

File: 3.11/graphs-pregel-algorithms.md (hunk 1 above)
Comment (romanatarango) [reply 3.11/graphs-pregel-algorithms.md:135]: I may be wrong, but as far as I know, after a colon, we start from the capital letter if what is before the colon refers to more than one sentence. In this case, we have two.

Comment (nerpaula) [reply 3.11/graphs-pregel-algorithms.md:135]: The style guide we use recommends using a lowercase for the first word that follows a colon. We can also drop the colon and have separate sentences.
https://developers.google.com/style/colons#colons-within-sentences 
https://developers.google.com/style/capitalization#capitalization-and-colons

**Pull request merged**